ci/build-test.cmd

```diff
@@ -0,0 +1,3 @@
+@echo off
+
+"%ProgramFiles%\git\bin\bash.exe" -c "./ci/build-test.sh"
```

Comment:
Might be a n00b question, but is this supposed to work on non-Windows laptops?

Reply:
nope, buildkite currently only has windows agents

Reply:
Got it. Do we intend to use buildkite for iOS/any other non-Windows builds at some point then?

Reply:
Yes - everything will be moved to Buildkite eventually

Reply:
yes, but it's not that easy, so focusing on windows first. but the plan is to completely move to buildkite at some point

Reply:
Makes sense